Workade/Views&ViewModels/Explore/RegionInfoView.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  RegionInfoView.swift
+//  Workade
+//
+//  Created by 김예훈 on 2022/11/22.
+//
+
+import UIKit
+
+class RegionInfoView: UIView {
```

Comment:
[P1] 더 이상 상속하지 않는다면 final 선언 부탁드립니다.
```suggestion
final class RegionInfoView: UIView {
```

Reply:
final을 자주 깜빡깜빡하네용..ㅠ 수정했습니다!!